docker/docker_nightly.py

```diff
@@ -54,18 +54,17 @@
     )
 
     # Push Nightly images to official PyTorch Dockerhub account
-    try_and_handle(f"docker push {organization}/{cpu_version}", dry_run)
     try_and_handle(f"docker push {organization}/{gpu_version}", dry_run)
     try_and_handle(f"docker push {organization}/{cpp_dev_cpu_version}", dry_run)
     try_and_handle(f"docker push {organization}/{cpp_dev_gpu_version}", dry_run)
 
     # Tag nightly images with latest
     try_and_handle(
-        f"docker tag {organization}/{cpu_version} {organization}/{project}:latest-cpu",
+        f"docker buildx imagetools create --tag {organization}/{project}:latest-cpu {organization}/{cpu_version}",
```

Comment:
Does this automatically push both the cpu images?

Reply:
Yes with the new tags of latest-cpu